ci/gpu/build.sh

```diff
@@ -62,7 +62,8 @@ conda install "rmm=$MINOR_VERSION.*" "cudatoolkit=$CUDA_REL" \
               "rapidjson" "flatbuffers" "boost-cpp" "fsspec>=0.6.0" "dlpack" \
               "feather-format" "cupy>=6.6.0,<8.0.0a0,!=7.1.0" "arrow-cpp=0.15.0" "pyarrow=0.15.0" \
               "fastavro>=0.22.0" "pandas>=0.25,<0.26" "hypothesis" "s3fs" "gcsfs" \
-              "boto3" "moto" "httpretty" "streamz" "ipython=7.3*" "jupyterlab"
+              "boto3" "moto" "httpretty" "streamz" "ipython=7.3*" "jupyterlab" \
+              "pytorch" "torchvision" "-c" "pytorch"
```

Comment:
cc @rapidsai/ops is this something already in the environment that we can use? Otherwise this would be big / expensive.

Reply:
It is not in any integration environment we have right now. But it should be added to the build environment if cudf is going to use it, CLX uses it as well.

Reply:
@galipremsagar lets remove this and work with ops to get it added into the base container.